packages/TesterApp/ios/TesterApp/AppDelegate.mm

```diff
@@ -26,10 +26,9 @@ - (NSURL *)bundleURL
   return [[NSBundle mainBundle] URLForResource:@"main" withExtension:@"jsbundle"];
 #endif
 }
-// Bridgeless mode is enabled by default in RN>=0.74 but we don't support it yet.
 - (BOOL)bridgelessEnabled
 {
-    return NO;
+    return YES;
 }
```

Comment:
I think this can be removed, right? As far as I understand there is no need to support new arch without bridgeless on 0.74 now

Reply:
yeah absolutely, just left it there for visibility so we can switch on demand for some edge cases